app/jobs/generate_book_plot_job.rb

```
@@ -3,16 +3,45 @@ class GenerateBookPlotJob < MetaJob
+      parameters: {
+        model: self.class.openai_model || ENV['OPENAI_MODEL'],
+        messages:,
+        temperature: 0.7,
+        response_format: { type: "json_object" }
+      }
+    )
+  end
+
   def update_component(response)
+    data = JSON.parse(response["choices"][0]["message"]["content"])
+    
```

RuboCop (1.69.1)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)